lib/util/foretoken.ex

```diff
@@ -0,0 +1,22 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule Antikythera.Foretoken do
```

Comment:
Should we name this module as something like `Antikythera.TokenBucket`?
I think we don't have to keep the name of the underlying library here.

Reply:
I think so.
If we use the same name as underlying library, it will be difficult to replace `Foretoken`.
So, I renamed.
Fixed in 07a9f4a74500d9fac41677ee52637e6db7f6965f